Makefile

```diff
@@ -11,14 +11,13 @@ endif
 LOCAL_BUILD_FLAGS ?= $(BUILD_FLAGS)
 LDFLAGS ?= -X=main.version=$(VERSION)
 DOCKERFILE = docker/Dockerfile
-IMAGE ?= pierone.example.com/acid/$(BINARY)
+IMAGE ?= registry.opensource.zalan.do/acid/$(BINARY)
```

Comment:
we need registry-write in order to push images

Reply:
It is overwritten in delivery.yaml, I just wanted to remove the pierone references.